### ISSUE TITLE
Please assign me a task

### ISSUE THREAD

**Issue by Nidita** [shivanshugarg12800/Algo_GT#66]
**Is your feature making change to an existing problem/algorithm**
answer in yes or no

**If you are making changes, please describe the type of change in brief else NA**

**Describe in brief about what new problem/algorithm addition**
A clear and concise description about the problem.

**Comment by Muskan4010** [comment]
I would like to add "Heap Sort in C++ using array" 
Will it be fine?

**Comment by shivanshugarg12800** [comment]
Yes @Muskan4010 you surely can add that, but do check if the problem exists previously or not.

**Comment by Muskan4010** [comment]
Thanks, I checked it already.
I have another doubt will this PR be counted for hacktoberfest?

**Comment by shivanshugarg12800** [comment]
Yes, surely

**Comment by Muskan4010** [comment]
Ok, Thanks a lot!
